packages/next/src/server/base-server.ts

```diff
@@ -928,11 +947,8 @@ export default abstract class Server<
       // it captures the initial URL.
       this.attachRequestMeta(req, parsedUrl)
 
-      let finished: boolean = false
-      if (this.minimalMode && this.enabledDirectories.app) {
```

Comment:
how come we removed these checks?

Reply:
It wasn't needed. The RSC handler uses feature detection using which normalizer, and in this case, they are added based on the presence of `this.minimalMode` and `this.enabledDirectories.app` already.